tests/integration/simple_copy_test/test_simple_copy.py

```diff
@@ -266,7 +266,8 @@ def test__snowflake__incremental_overwrite(self):
         # Setting the incremental_strategy should make this succeed
         self.use_default_project({
             "models": {
-                "incremental_strategy": "delete+insert"
+                "incremental_strategy": "delete+insert",
+                "tmp_relation_type": "table"
```

Comment:
Is this change necessary? Or was it just temporary and can be restored back to the original now?
```suggestion
                "incremental_strategy": "delete+insert"
```

Reply:
removed config testing locally and will push up